prov/verbs/src/verbs_info.c

```diff
@@ -890,18 +890,45 @@ static int fi_ibv_getifaddrs(struct dlist_entry *verbs_devs)
 	const char *ret_ptr;
 	int ret, num_verbs_ifs = 0;
 
+	char *iface = 0;
+	char iface_len = 0;
```

Comment:
there is limit for iface name length = 16 bytes, that is why I used char... but you are right - size_t is better here.
fixed. thank you